lib/cdo/pegasus.rb

```diff
@@ -7,7 +7,6 @@
 require_relative './pegasus/hash'
 require_relative './pegasus/object'
 require_relative './pegasus/string'
-require_relative './pegasus/only_one_running'
```

Comment:
Unless you disagree, I would prefer to manually insert the dependency in all the places where it is used. Planning on following up with such another commit.